Neos.Utility.ObjectHandling/Tests/Unit/TypeHandlingTest.php

```diff
@@ -202,7 +202,8 @@ public function collectionTypes()
             ['ArrayObject', true],
             ['SplObjectStorage', true],
             ['Doctrine\Common\Collections\Collection', true],
-            ['Doctrine\Common\Collections\ArrayCollection', true]
+            ['Doctrine\Common\Collections\ArrayCollection', true],
+            ['IteratorAggregate', true]
```

Comment:
```suggestion
            ['IteratorAggregate', true],
            ['Iterator', true]
```
If we do the above, we can add a test for `\Iterator` too